pkg/debug/spec.sh

```diff
@@ -161,7 +161,7 @@ if [ -e /dev/xen ]; then
    MEM=$(( $(eve exec xen-tools xl info | grep total_memory | cut -f2 -d:) ))
 else
    CPUS=$(grep -c '^processor.*' < /proc/cpuinfo)
-   MEM=$(awk '/MemTotal:/ { print int($2 / 1024); }' < /proc/meminfo)
```

Comment:
This can cause significant truncation for devices which only have a few GB of memory, since Linux typically doesn't get all of the 2G/4G/8G with the BIOS stealing some of it.
Hence you'll truncate to 1G/3G/7G which is a significant loss.

Was there a reason to truncate to GB?

Reply:
This is the reason to be in KB: https://github.com/lf-edge/eve/commit/7ef8cd37896b3f03685209edf70533ba15cc8dfa

@christoph-zededa , which errors are you referring to? I didn't experience any error while testing these changes.

Reply:
@rene I agree the unit should be in MB, but appearently there is a bug in zcli which can't handle the trailing "M" but if you give it a number it is correctly interpreted as being in MB.

Reply:
GB causes too much loss of accuracy. Hence we should keep this in MB.
But need to drop the 'M' at the end of the memory number since zcli is currently broken if you add that.